### PR TITLE
Ensure QUIT messages are sent in pooling configuration

### PR DESCRIPTION
**Pull request by Half-Shot**
Failing to specify a readyState means we missed this one.

### CODE REVIEW

File: spec/e2e/quit.spec.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            matrixLocalparts: ['alice'],
+            ircNicks: ['bob'],
+            traceToFile: true,
+        });
+        await testEnv.setUp();
+    });
+    afterEach(() => {
+        return testEnv?.tearDown();
+    });
+    it('should correctly send a quit message on ', async () => {
```
Comment (tadzik) [inline spec/e2e/quit.spec.ts]:
```suggestion
    it('should correctly send a quit message on !reconnect', async () => {
```

File: src/pool-service/RedisIrcConnection.ts [hunk 2]
```diff
@@ -11,12 +11,18 @@ export type RedisIrcConnectionEvents = {
...
 export class RedisIrcConnection extends (EventEmitter as unknown as
     new () => TypedEmitter<RedisIrcConnectionEvents&IrcConnectionEventsMap>) implements IrcConnection {
-    private readonly log = new Logger(`RedisIrcConnection:${this.clientId}`);
+    private readonly log: Logger;
 
     public get connecting() {
         return this.isConnecting;
     }
 
+    public get readyState() {
```
Comment (tadzik) [inline src/pool-service/RedisIrcConnection.ts]: It'd be nice to have this typed, but seems like it's not typed in node-irc either :/ Next time.

**Review by tadzik — APPROVED** [review]
Just nitpicks.

File: src/pool-service/RedisIrcConnection.ts (hunk 2 above)
Comment (Half-Shot) [reply src/pool-service/RedisIrcConnection.ts]: I can type it here at least.

**Pull request merged**